docs/solution-architecture/accessControlPolicies.md

```diff
@@ -45,68 +45,68 @@ Finally, there is a special `domein_elk_zaaktype` system role which will grant t
 The following OPA access control policies are enforced in the ZAC backend for the roles listed above for the
 various resources on which a user can perform actions:
 
```

Comment:
Annoying that this table is reformatted in PRs. Makes reviewing harder. But oh well.